### PR TITLE
Updating style guide to include inclusive language

### PR DESCRIPTION
**Pull request by Rumyra**
> What was wrong/why is this fix needed? (quick summary only)

Whilst we have a guideline for gender neutral language, there is nothing about using inclusive language, this adds a better section for that

### CODE REVIEW

File: files/en-us/mdn/guidelines/writing_style_guide/index.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h5 id="Code_Examples">Code Examples</h5>
 <p>The text following each piece of code should explain anything relevant, using an appropriate level of detail:</p>
 
 <ul>
- <li>If the code is very simple and doesn't really use directly anything related to the API being documented, you may only give a quick summary of what it is and why it's there.</li>
+ <li>If the code is very simple and doesn't really use anything directly related to the API being documented, you may only give a quick summary of what it is and why it's there.</li>
```
Comment (hamishwillee) [inline files/en-us/mdn/guidelines/writing_style_guide/index.html:132]: "may only" or " need only"?

File: files/en-us/mdn/guidelines/writing_style_guide/index.html [hunk 2]
```diff
@@ -426,9 +426,23 @@ <h3 id="Hyphenation">Hyphenation</h3>
...
+
+<p>MDN has a wide and diverse audience. We strongly encourage keeping text as inclusive as possible. Here are some alternatives to common terms used in documentation:</p>
+
+<ul>
+  <li>Avoid using the terms <b>master</b> and <b>slave</b> and instead <b>main</b> and <b>replica</b></li>
+  <li>Replace <b>whitelist</b> and <b>blacklist</b> with <b>allowlist</b> and <b>denylist</b></li>
+  <li><b>Sanity</b> should be replaced with <b>coherence</b></li>
+  <li>Instead of <b>dummy</b> you should use <b>placeholder</b>
+  <li>You should not need to use <b>crazy</b> and <b>insane</b> in documentation, however if the case arises consider <b>fantastic</b> instead.</li>
+
```
Comment (hamishwillee) [inline files/en-us/mdn/guidelines/writing_style_guide/index.html:440]: Unnecessary space.

File: files/en-us/mdn/guidelines/writing_style_guide/index.html [hunk 3]
```diff
@@ -426,9 +426,23 @@ <h3 id="Hyphenation">Hyphenation</h3>
...
+<h3 id="Inclusive_language">Inclusive language</h3>
+
+<p>MDN has a wide and diverse audience. We strongly encourage keeping text as inclusive as possible. Here are some alternatives to common terms used in documentation:</p>
+
+<ul>
+  <li>Avoid using the terms <b>master</b> and <b>slave</b> and instead <b>main</b> and <b>replica</b></li>
+  <li>Replace <b>whitelist</b> and <b>blacklist</b> with <b>allowlist</b> and <b>denylist</b></li>
+  <li><b>Sanity</b> should be replaced with <b>coherence</b></li>
+  <li>Instead of <b>dummy</b> you should use <b>placeholder</b>
+  <li>You should not need to use <b>crazy</b> and <b>insane</b> in documentation, however if the case arises consider <b>fantastic</b> instead.</li>
```
Comment (hamishwillee) [inline files/en-us/mdn/guidelines/writing_style_guide/index.html:439]: FWIW it took me a second to realise that these were being used as superlatives. I'm not sure worth "fixing".

**Review by hamishwillee — APPROVED** [review]

File: files/en-us/mdn/guidelines/writing_style_guide/index.html (hunk 3 above)
Comment (teoli2003) [reply files/en-us/mdn/guidelines/writing_style_guide/index.html:439]: I think these will be "fixed" when converting to md.

**Comment by rachelandrew** [comment]
I wonder if it is worth adding links to other styleguides, here, showing that this isn't something MDN are taking on as a cause, but an alignment with good practice for writing that has been adopted by other major styleguides. For example:

- https://developers.google.com/style/inclusive-documentation?hl=en the Google guide has some great examples of replacement common phrases. Including what to do if the thing you are writing about using non-inclusive terms.
- https://docs.microsoft.com/en-us/style-guide/bias-free-communication

File: files/en-us/mdn/guidelines/writing_style_guide/index.html [hunk 4]
```diff
@@ -581,7 +596,7 @@ <h3 id="Preferred_dictionary">Preferred dictionary</h3>
 
 <p>For questions of spelling, please refer to <a href="https://www.dictionary.com/">Dictionary.com</a>. The spelling checker for this site uses American English. Please do not use variant spellings (e.g., use <em>color</em> rather than <em>colour</em>).</p>
 
-<p>We will be expanding the guide over time, so if you have specific questions that aren't covered in this document, please post them on the <a href="https://discourse.mozilla.org/c/mdn">MDN discussion forum</a>, so we know what should be added.</p>
+<p>We will be expanding the guide over time, so if you have specific questions that aren't covered in this document, <a href="https://developer.mozilla.org/en-US/docs/MDN/Contribute/Getting_started#step_4_ask_for_help">please get in touch</a>, so we know what should be added.</p>
```
Comment (Elchi3) [inline files/en-us/mdn/guidelines/writing_style_guide/index.html:599]:
```suggestion
<p>We will be expanding the guide over time, so if you have specific questions that aren't covered in this document, <a href="/en-US/docs/MDN/Contribute/Getting_started#step_4_ask_for_help">please get in touch</a>, so we know what should be added.</p>
```

**Review by Elchi3 — APPROVED** [review]

File: files/en-us/mdn/guidelines/writing_style_guide/index.html (hunk 3 above)
Comment (Rumyra) [reply files/en-us/mdn/guidelines/writing_style_guide/index.html:439]: I specifically made that decision as there's no emphasis around the words, only styling was needed.

**Comment by Ryuno-Ki** [comment]
@Rumyra In case you prepare an update, can we include https://www.selfdefined.app/ ?
According to the motto „don't talk about us without us” I would like to defer to a dictionary maintained by the group which it is about.

Regarding inclusive language in tech, https://css-tricks.com/words-avoid-educational-writing/ inspired Zach to write an inclusive language plugin for Eleventy (but it uses plain regular expression, i.e. is unaware of context).

**Pull request merged**